core/src/main/scala/kafka/admin/ConfigCommand.scala

```diff
@@ -64,23 +60,14 @@ import scala.collection._
  * Alternatively, --user-defaults, --client-defaults, --broker-defaults, or --ip-defaults may be specified in place of
  * --entity-type <users|clients|brokers|ips> --entity-default, respectively.
  *
- * For most use cases, this script communicates with a kafka cluster (specified via the
- * `--bootstrap-server` option). There are three exceptions where direct communication with a
- * ZooKeeper ensemble (specified via the `--zookeeper` option) is allowed:
- *
- * 1. Describe/alter user configs where the config is a SCRAM mechanism name (i.e. a SCRAM credential for a user)
- * 2. Describe/alter broker configs for a particular broker when that broker is down
- * 3. Describe/alter broker default configs when all brokers are down
- *
- * For example, this allows password configs to be stored encrypted in ZK before brokers are started,
- * avoiding cleartext passwords in `server.properties`.
+ * For most use cases, this script communicates with a Kafka cluster (specified via the
+ * `--bootstrap-server` option).
```

Comment:
I think we can remove these 2 lines as the tool no longer communicates with ZooKeeper

Reply:
That make sense, thanks for your review again!